.github/workflows/deploy-infra.yaml

```diff
@@ -200,12 +200,38 @@ jobs:
         continue-on-error: true
         run: |-
           kubectl create namespace flux
-      
+
+      - id: install-yq
+        name: Install yq for modifying the deployment spec.
+        run: |-
+          sudo wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/local/bin/yq && sudo chmod +x /usr/local/bin/yq
+
+      - id: fill-account-specific-metadata
+        name: Fill in account specific metadata in ConfigMap
+        run: |-
+          if [ "${{ matrix.environment }}" = "production" ]; then
+            DOMAIN_NAME="scp.biomage.net"
+          fi
+
+          if [ "${{ matrix.environment }}" = "staging" ]; then
+            DOMAIN_NAME="scp-staging.biomage.net"
+          fi
```

Comment:
The domain names are still hard-coded here. Does that mean that when this repo is forked, these hardcoded values have to also be changed in the forked repo? Why not save it in a Github environment variable?

Reply:
we will fix the hardcoded domain name in one of the next tickets (see the description). As an initial iteration, I have made it hardcoded.